src/astro/utils/dataframe_function_handler.py

```diff
@@ -0,0 +1,95 @@
+import inspect
+from abc import ABC
+from typing import Callable, Dict, Optional, Tuple
+
+import pandas as pd
+
+from astro.databases import create_database
+from astro.databases.base import BaseDatabase
+from astro.sql.table import Table
+
+
+class DataframeFunctionHandler(ABC):
```

Comment:
@dimberman Why do we need this in the context of transform?

Reply:
@tatiana I'm following @kaxil 's request that we separate out functions in case we want to re-use them later.